lib/features/support/env.rb

```diff
@@ -167,14 +177,15 @@ def read_key_path hash, key_path
 class Servlet < WEBrick::HTTPServlet::AbstractServlet
   def do_POST request, response
     case request['Content-Type']
-    when 'application/json'
-      stored_requests << {body: JSON.load(request.body()), request:request}
     when /^multipart\/form-data; boundary=([^;]+)/
       boundary = WEBrick::HTTPUtils::dequote($1)
       body = WEBrick::HTTPUtils.parse_form_data(request.body(), boundary)
       stored_requests << {body: body, request: request}
     else
-      stored_requests << {body: request.query, request:request}
```

Comment:
I'm fairly certain this logic isn't used anymore and was replaced by the multipart parser, though @fractalwrench may want to double check on the gradle plugin.

Reply:
lgtm